docs/changelog.md

```diff
@@ -6,6 +6,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 ## [1.4.0] -- 2023-04-24
 
 ### Added
+
```

Comment:
Can you add updates to changelog? I guess this is beginning of 1.5.0, you can leave the date off